TYPO3.Media/Classes/TYPO3/Media/Package.php

```diff
@@ -13,7 +13,7 @@
 
 use TYPO3\Flow\Core\Bootstrap;
 use TYPO3\Flow\Package\Package as BasePackage;
-use TYPO3\Media\Domain\Model\Asset;
+use TYPO3\Media\Domain\Service\AssetService;
```

Comment:
Ok, it is used, but in the `connect()` call will resolve to a string just fine. Only the connectd signal will never fire… :)